core/dslmcode/shared/drupal-7.x/modules/elmsln_contrib/cis_connector/modules/features/elmsln_core/elmsln_core.module

```diff
@@ -95,6 +95,35 @@ function elmsln_core_menu() {
 }
 
 /**
+ * Implements hook_menu_alter.
+ * Send user to the CPR to edit the global profile.
+ */
+function elmsln_core_menu_alter(&$items) {
+
+  //setup default user profile repository so it can be overriden by a global conf variable.
+  $elmsln_global_profile = variable_get('elmsln_global_profile', $default = 'cpr');
+
+  if(variable_get('install_profile') != $elmsln_global_profile) {
```

Comment:
always supply a default w/ variable_get; could just do this and 104 inline; w/ profile default as standard since that's the convention used throughout core
```
if (variable_get('install_profile', 'standard') != variable_get('elmsln_global_profile', 'cpr')) {
```